src/component/DiaryHome.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Box from "@material-ui/core/Box";
+import TextField from "@material-ui/core/TextField";
+import TextareaAutosize from "@material-ui/core/TextareaAutosize";
+import Button from "@material-ui/core/Button";
+
+export default function DiaryHome() {
+  const classes = useStyles();
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+  const onClick = (e) => {
+    console.log("title=" + title + " description=" + description);
+    e.preventDefault();
+    setTitle("");
+    setDescription("");
+  };
+
+  return (
+    <Box p={2} bgcolor="background.paper">
+      <form className={classes.root} noValidate autoComplete="off">
+        <TextField
+          underlineShow={true}
+          color="white"
+          id="Title"
+          placeholder="Submit New"
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+          inputStyle={{ color: "white", padding: "0 15px" }}
+          style={{ background: "#fed8b1", borderRadius: 25 }}
+        />
+        <br />
```

Comment:
Use margins and paddings instead of br tags.

Reply:
Done